recurring_contract/models/contract_group.py

```diff
@@ -240,7 +240,8 @@ def _generate_invoices(self):
             # Compute the interval of months there should be between each invoice (set in the contract group)
             recurring_unit = pay_opt.recurring_unit
             month_interval = pay_opt.recurring_value * (1 if recurring_unit == "month" else 12)
-            for inv_no in range(1, pay_opt.advance_billing_months + 1, month_interval):
+            curr_month = self.env["ir.config_parameter"].sudo().get_param(f"compassion_nordic_accounting.do_generate_curr_month_{self.env.company.id}", 0)
+            for inv_no in range(0 if curr_month else 1, pay_opt.advance_billing_months + 1, month_interval):
```

Comment:
`def get_param` is declared in module `message_center_compassion` and is thus not accessible from within this module that has no dependency on it.

Can the company be read from the environment or should it be taken from the contract_group or contract object?

```suggestion
            param_string = f"compassion_nordic_accounting.do_generate_curr_month_{self.env.company.id}"
            curr_month = self.env["ir.config_parameter"].sudo().default_get([param_string]).get(param_string, False)
            for inv_no in range(0 if curr_month else 1, pay_opt.advance_billing_months + 1, month_interval):
```